server/zoom/common/constants.py

```diff
@@ -3,11 +3,7 @@
 
 __env_connections = {
     "local": "localhost:2181",
-    "Staging": ('ZooStaging01:2181,'
-                'ZooStaging02:2181,'
-                'ZooStaging03:2181,'
-                'ZooStaging04:2181,'
-                'ZooStaging05:2181'),
+    "Staging": ('zk:2181'),
```

Comment:
This will need to be backed out. It's left over from your using the unreleased dockerized version.